tests/test_examples.py

```diff
@@ -164,11 +170,13 @@ def _run_uninstaller_exe(install_dir, timeout=420, check=True):
         "start",
         "/wait",
         str(uninstaller),
+        "/S",
+        *options,
```

Comment:
I think the native NSIS flags like `/S` need to go AFTER the custom ones.

Reply:
I think the order is "options without arguments before options with arguments".

When I ran the tests, I noticed that `/RemoveCondaRcs=user /S`, `${GetOptions} $ARGV "/RemoveCondaRcs=" $ARGV_Uninst_RemoveCondaRcs` will set `$ARGV_Uninst_RemoveCondaRcs` to `user "/S`.

Reply:
Got it. Can we reflect these findings in the docs? I think there's a section dedicated to this aspect.

Reply:
I updated and expanded the documentation